pkg/cloud/aws/bootstrap/pod.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: aws-cloud-controller-manager
+  namespace: kube-system
+spec:
+  priorityClassName: system-cluster-critical
+  containers:
+  - args:
+    - --cloud-provider=aws
+    - --use-service-account-credentials=false
+    - --controllers=*,-service,-route # disable service and route controllers in bootstrap
+    - --kubeconfig=/etc/kubernetes/secrets/kubeconfig
+    - --leader-elect=false
```

Comment:
Might it cause conflicts when ccm from actual masters comes in?

Reply:
Saw some failures with custom resource used for leader election here, so this feature is disabled. No it would not, additionally CCM could run in parallel with little or no issues. Once all masters are ready this bootstrap node would go away, and so additional CCM replica.